common/views/components/License/License.js

```diff
@@ -0,0 +1,51 @@
+import getLicenseInfo from '../../../utils/get-license-info';
+import {spacing, font} from '../../../utils/classnames';
+import Icon from '../Icon/Icon';
+
+function renderAboutAttr(subject) {
+  if(subject) {
+    return `about='${subject}`;
+  }
+}
```

Comment:
Why not just stick this in the component, the logic is super simple and we only use it in 1 place?